apparmor.d/groups/ssh/sshd

```diff
@@ -73,17 +76,17 @@ profile sshd @{exec_path} flags=(attach_disconnected) {
         @{run}/motd.dynamic.new rw,
         @{run}/resolvconf/resolv.conf r,
         @{run}/systemd/sessions/[0-9]*.ref rw,
-        @{run}/systemd/userdb/ r,
+        @{run}/systemd/notify w,
 
   @{sys}/fs/cgroup/*/user/*/[0-9]*/ rw,
   @{sys}/fs/cgroup/systemd/user.slice/user-@{uid}.slice/session-c[0-9]*.scope/ rw,
 
-  owner @{PROC}/@{pid}/limits r,
-  owner @{PROC}/@{pid}/loginuid rw,
-  owner @{PROC}/@{pid}/mounts r,
-  owner @{PROC}/@{pid}/oom_adj rw,
-  owner @{PROC}/@{pid}/oom_score_adj rw,
-  owner @{PROC}/@{pid}/uid_map r,
+  owner @{PROC}/@{pids}/limits r,
```

Comment:
We are switching to `@{pids}`, right?

Reply:
No we are not, `@{pid}` and `@{pids}` will in the future not be the same thing. `@{pid}` refers to the pid of the profile. In practice, I usually putt `@{pid}` with `owner` and  `@{pids}` without it.

Reply:
Got it, reverted.